src/lib.rs

```diff
@@ -41,6 +39,8 @@ pub enum Operation {
   Delete,
 }
 
+const SYNTHETIC_EOF: &[u8] = b"\x1B]7878\x1B\\";
```

Comment:
y u no


```suggestion
#[napi]
const SYNTHETIC_EOF: &[u8] = b"\x1B]7878\x1B\\";
```

?

if napi doesn't like this, export a function that returns this byte array, that _should_ definitely work.